packages/react/src/types.ts

```diff
@@ -31,5 +31,65 @@ export type CssObject<TProps> = Readonly<{
 export type CssFunction<TProps = unknown> =
   | CssType<TProps>
   | BasicTemplateInterpolations // CSS values in tagged template expression
+  | null
   | boolean // Something like `false && styles`
   | undefined; // Something like `undefined && styles`
+
+// List of pseudo-classes and pseudo-elements are from csstype
+// but with & added in the front, so that we target the current element
+// (instead of a child element)
```

Comment:
I originally wrote this comment and it's wrong haha, both `:hover` and `&:hover` target the current element

Didn't DST decide to go with the `&`-less syntax in the end? I don't think it's too late to change our syntax as nobody is using cssMap yet

Reply:
We're flexible. If we lean on `&` it is at least following the nested spec which means we can do less transformations.

I'd say let's do whatever the nested spec does.

Reply:
Ah all good, `cssMap` is also pretty flexible (the restrictions on using `&:hover` instead of `:hover` are pretty arbitrary) so feel free to make changes to that if needed

Reply:
In my past, I've seen `& :hover` meaning "self child:hover" so `:hover` could mean "child:hover" while `&:hover` explicitly means "self:hover", so my vote is for `&:pseudo`

Reply:
Yeah exactly. We'll keep it as is.